docs/snippets/vpc-cni-custom-networking.md

```diff
@@ -15,77 +15,113 @@ In this example, the `vpc-cni` addon is configured using `before_compute = true`
 
 If you find that your nodes are not being created with the correct number of max pods (i.e. - for `m5.large`, if you are seeing a max pods of 29 instead of 110), most likely the `vpc-cni` was not configured *before* the EC2 instances.
 
+To be able to do this configuration you need access to your cluster from where you are deploying the terraform code. In case it is a private cluster access the VPC through a VPN, Direct Connect or Bastion, and ensure the entity used has enough permissions in the cluster. 
 ## Components
 
 To enable VPC CNI custom networking, you must configuring the following components:
 
 1. Create a VPC with additional CIDR block associations. These additional CIDR blocks will be used to create subnets for the VPC CNI custom networking:
 
-      ```json
+      ```
       module "vpc" {
-      source  = "terraform-aws-modules/vpc/aws"
+         source  = "terraform-aws-modules/vpc/aws"
```

Comment:
if you copy+paste this code locally and run `terraform fmt .` - what does it look like? (hint: it will be left aligned and not indented like this)